### PR TITLE
add readme improvements

### PR DESCRIPTION
**Pull request by aniketmaurya**
<details>
  <summary><b>Before submitting</b></summary>

- [ ] Was this discussed/agreed via a Github issue? (no need for typos and docs improvements)
- [ ] Did you read the [contributor guideline](https://github.com/Lightning-AI/pytorch-lightning/blob/main/.github/CONTRIBUTING.md), Pull Request section?
- [ ] Did you make sure to update the docs?
- [ ] Did you write any new necessary tests?

</details>

## What does this PR do?

- [x] Automate sanity check for README codes
- [x] Every code must be runnable
- [x] imports at the top
- [x] Be explicit and don't use "..."

## PR review

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in GitHub issues there's a high chance it will not be merged.

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -367,7 +367,7 @@ and implement `LitAPI.predict` to process batched inputs.
 import numpy as np
 import litserve as ls
 
-class SimpleStreamAPI(ls.LitAPI):
+class SimpleBatchedAPI(ls.LitAPI):
```
Comment (aniketmaurya) [inline README.md:370]: sorry this was misnamed by me in #65

File: README.md [hunk 2]
```diff
@@ -290,23 +290,27 @@ class SimpleLitAPI(ls.LitAPI):
...
 ```
 
 The `devices` argument can also be an array specifying what device id to
 run the model on:
 
 ```python
-server = LitServer(SimpleLitAPI(), accelerator="cuda", devices=[0, 3])
+import litserve as ls
+
+server = LitServer(ls.examples.SimplePyTorchAPI(), accelerator="cuda", devices=[0, 3])
```
Comment (lantiga) [inline README.md:303]:
```suggestion
server = LitServer(ls.examples.SimpleTorchAPI(), accelerator="cuda", devices=[0, 3])
```

File: README.md [hunk 3]
```diff
@@ -290,23 +290,27 @@ class SimpleLitAPI(ls.LitAPI):
...
 
 Last, you can run multiple copies of the same model from the same device,
 if the model is small. The following will load two copies of the model on
 each of the 4 GPUs:
 
 ```python
-server = LitServer(SimpleLitAPI(), accelerator="cuda", devices=4, workers_per_device=2)
+import litserve as ls
+
+server = LitServer(ls.examples.SimplePyTorchAPI(), accelerator="cuda", devices=4, workers_per_device=2)
```
Comment (lantiga) [inline README.md:313]:
```suggestion
server = LitServer(ls.examples.SimpleTorchAPI(), accelerator="cuda", devices=4, workers_per_device=2)
```

**Review by lantiga — CHANGES_REQUESTED** [review]
I would actually make the import explicit so the code remains cleaner

```
from litserve.examples import SimpleLitAPI
```

**Comment by aniketmaurya** [comment]
CI should pass after #73

**Comment by aniketmaurya** [comment]
@lantiga windows CI seems to hang for README code pytest.

**Review by lantiga — APPROVED** [review]

**Pull request merged**